app/src/main/java/de/danoeh/antennapod/activity/OpmlImportBaseActivity.java

```diff
@@ -19,22 +30,23 @@
     private static final String TAG = "OpmlImportBaseActivity";
     private OpmlImportWorker importWorker;
 
-    /**
+	private static final int PERMISSION_REQUEST_READ_EXTERNAL_STORAGE = 5;
```

Comment:
Do we really have to define this ourselves?  That's seems terrible (but I can't find it anywhere else)